.github/auto-labeler.yml

```diff
@@ -0,0 +1,21 @@
+language/go:
+  - changed-files:
+      - any-glob-to-any-file:
+          - "*.go"
+          - "**/*.go"
+
+area/docs:
+  - changed-files:
+      - any-glob-to-any-file:
+          - "*.md"
+          - "docs/**"
+
+area/ci:
+  - changed-files:
+      - any-glob-to-any-file:
+          - ".github/workflows/**"
+
+area/templates:
```

Comment:
One thing to be aware of is that each of these sections must match an existing [issue label](https://github.com/meshery-extensions/meshery-linkerd/labels).